betterproto/tests/test_features.py

```diff
@@ -263,3 +263,14 @@ class TestParentMessage(betterproto.Message):
         "someDouble": 1.2,
         "someMessage": {"someOtherInt": 0},
     }
+
+
+def test_oneof_default_value_set_causes_writes_wire():
+    @dataclass
+    class Foo(betterproto.Message):
+        bar: int = betterproto.int32_field(1, group="group1")
+        baz: str = betterproto.string_field(2, group="group1")
+
+    assert bytes(Foo(bar=0)) != b""
+    assert bytes(Foo(baz="")) == b""  # Baz is just an empty string
```

Comment:
Does this mean its not possible to detect `which_one_of` if one of the groups is a `String` and the value was the empty string?

Same with other default values? 😕

Reply:
Good catch - this actually works with the other default values but not for a string at the moment, I'll get a fix in 😄